models/prices/prices_unsupported_chains_non_native_tokens.sql

```diff
@@ -0,0 +1,25 @@
+{{ config(
+        schema='prices_unsupported_chains_non_native',
+        alias = 'tokens',
+        materialized='table',
+        file_format = 'delta',
+        tags = ['static']
+        )
+}}
+SELECT 
+    token_id
+    , blockchain
+    , symbol
+    , contract_address
+    , CAST(decimals as int) as decimals
+FROM
+(
+    VALUES
+    ('btcs-ordinals', 'brc-20', 'BTCS', 'edc052335f914ee47a758cff988494fbb569d820e66ac8581008e44b26dcdb43i0', null),
```

Comment:
My understanding of ordinals aren't great but aren't they techincally on bitcoin? If so we should just add these in bitcoin.